.changeset/eleven-emus-warn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@definitelytyped/dtslint": patch
+---
+
+remove `aws-lambda` from expected npm version failures
```

Comment:
Is this file pulled from the repo or from the package? If from the repo, we should update the workflow to not require this.

Reply:
I think it's from the package right now. Would be nice to do it from the repo.

Reply:
Going through my email backlog from being out; I'll send a PR for this quick!

Reply:
Sent https://github.com/microsoft/DefinitelyTyped-tools/pull/959